feedback_report_generator/README.md

```diff
@@ -0,0 +1,25 @@
+create feedback reports
+=================
+Program to create feedback reports.
+
+### Configuration Settings
+
+Before running the script, set the following environment variables.
+
+| Variable                            | Default                | Description                                              |
+| ----------------------------------- | ---------------------- | -------------------------------------------------------- |
+| AzureCosmosDbDatabaseId | retrieve from portal | The id of a CosmosDB instance that holds the latest ukrlp data in a container called urkrlp.|
+| AzureCosmosDbUkRlpCollectionId | feedback | the container name holding the ukrlp data |
+| AzureCosmosDbUri | retrieve from portal | the uri to CosmosDB |
+| AzureCosmosDbKey | retrieve from portal | the key to authenticate to CosmosDB |
+
+
+### Setup
+
+* Ensure python 3.6 ot greater is installed
+* Create a Python virtualenv to run this tool in
+* Activate the virtualenv
+
+```
+pip install -r requirements.txt
+```

```

Comment:
You haven't explained how to run the script. I did the following:

1) Add environment variables to `env.sh` file, like so:
```
export AzureCosmosDbUri="<uri>"
export AzureCosmosDbKey="<connection_key>"
export AzureCosmosDbDatabaseId="<database_id>"
export AzureCosmosDbFeedbackCollectionId="<collection_id>"
```
2) Run `source env.sh`
3) Run `python report_generator.py`

Reply:
👍